citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityEntryController.java

```diff
@@ -164,4 +200,12 @@
         return new ResponseEntity<IAuthorityEntry>(entry, HttpStatus.OK);
     }
 
+    private <T> List<T> getPage(List<T> entries, int page, int pageSize) {
```

Comment:
is there a reason this is generic?

Reply:
Doesn't have any other use over here. But it seemed that method could be a generic one.